Makefile

```diff
@@ -0,0 +1,7 @@
+SHELL:=/bin/bash
+
+black:
```

Comment:
Nit: could rename to something like `format` which more clearly conveys the effect of the command?

Reply:
not to be confused with formatting your system :0. I'll think of a generic name

Reply:
Also a "nit" shouldn't block a PR.